test/Microsoft.AspNetCore.Server.KestrelTests/BadHttpRequestTests.cs

```diff
@@ -103,10 +103,9 @@ public class BadHttpRequestTests
             await connection.Receive(
                 "Connection: close",
                 "");
-            await connection.ReceiveStartsWith("Date: ");
             await connection.ReceiveEnd(
+                $"Date: {connection.Server.Context.DateHeaderValue}",
                 "Content-Length: 0",
-                "Server: Kestrel",
```

Comment:
Are there any remaining tests the verify that the "Server: Kestrel" header is sent by default?